8/debian-10/rootfs/app-entrypoint.sh

```diff
@@ -45,6 +45,18 @@ replace_in_file() {
 # Returns: none
 #########################
 wait_for_db() {
+    local -r db_connection="${DB_CONNECTION:-mysql}"
```

Comment:
I'd recommend to rename this environment variable to `DB_TYPE` to be consistent with what we are using in other container images:

```suggestion
    local -r db_connection="${DB_TYPE:-mysql}"
```

Reply:
That env var appears to be from [laravel](https://laravel.com/docs/8.x/database#sqlite-configuration)

Reply:
Good point!